### PR TITLE
Convert into PWA

### LINKED ISSUE #12: Convert into PWA app
- Convert the web application into a PWA app that shows users the option to install it on their mobiles.

### LINKED ISSUE #18: Pick title and update logo favicon
Sooner or later we are going to have to come up with a neat name and add a custom favicon.

Any suggestions?

### PR DESCRIPTION
**Pull request by Pratyush1606**
Fixes #12 
* Converted the React App into a Progressive Web App
* Attached the PWA Summary Report

### CODE REVIEW

**Comment by Pratyush1606** [comment]
Attaching Progressive Web App Summary
[pwa_summary.pdf](https://github.com/MLH-Fellowship/prep-project-4.1.4/files/7575385/pwa_summary.pdf)

File: src/service-worker.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
+/* eslint-disable no-restricted-globals */
```
Comment (EkjotKaur) [inline src/service-worker.js:1]: You could add these two service files to a folder.

**Comment by EkjotKaur** [comment]
You can look at my suggested changes. The rest looks good to me.

**Comment by NoahCardoza** [comment]
Everything looks good! Can you fix the merge conflicts and then we are set!

**Review by EkjotKaur — COMMENTED** [review]
This is my review

File: src/service-worker.js (hunk 1 above)
Comment (NoahCardoza) [reply src/service-worker.js:1]: Good idea!

Comment (Pratyush1606) [reply src/service-worker.js:1]: These service-worker files are expected to be at the root level to control requests from all files in this domain. Also, these are generated by default at this level only.
Reference: [Introduction to Service Worker](https://developers.google.com/web/ilt/pwa/introduction-to-service-worker)

Comment (NoahCardoza) [reply src/service-worker.js:1]: Good catch @Pratyush1606 !

**Review by NoahCardoza — APPROVED** [review]
Looks good!

**Pull request merged**